src/repo-query/components/Dialog.tsx

```diff
@@ -0,0 +1,123 @@
+import { useState } from "react";
+import openSaucedLogoIcon from "../../assets/open-sauced-orange-bg-logo.svg";
+import { Home } from "../pages/home";
+import { IndexingPage } from "../pages/indexing";
+import { Chat } from "../pages/chat";
+import { RepoQueryPages } from "../../ts/types";
+
+export const Dialog = ({ isOpen, toggleDialog, ownerName, repoName }: { isOpen: boolean, toggleDialog: () => void, ownerName: string, repoName: string }) => {
+    const [currentPage, setCurrentPage] = useState(RepoQueryPages.Home);
+
+    return (
+        <div
```

Comment:
I haven't run this PR's code, but you can probably use the native [dialog element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) instead of a div.

Reply:
![image](https://github.com/open-sauced/ai/assets/41837037/436fd149-0a1a-4004-addd-f4a7d3dc4f20)
I might have to change some other things with this too, let's just keep it a div and change it later if we need to 😅